debian/rules

```diff
@@ -1,5 +1,27 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
+DEBIAN_DIST := $(shell lsb_release -ds | tr -d '()' | sed -e 's/\#/ /g')
+
+ifneq (,$(findstring Endless,$(DEBIAN_DIST)))
+debian/runtime-org.freedesktop.Platform.openh264.conf: debian/runtime-org.freedesktop.Platform.openh264.conf.in
+	### configure
+	sed -e 's/@DEB_HOST_MULTIARCH@/$(DEB_HOST_MULTIARCH)/g' \
+		$< > $@
+
+override_dh_install: debian/runtime-org.freedesktop.Platform.openh264.conf
+	mkdir -p debian/libnoopenh264-6/etc/ld.so.conf.d/
+	install -m 755 debian/runtime-org.freedesktop.Platform.openh264.conf debian/libnoopenh264-6/etc/ld.so.conf.d/
+	dh_install
+
+override_dh_clean:
+	rm -f debian/runtime-org.freedesktop.Platform.openh264.conf
+else
+# Do not install /etc/ld.so.conf.d and flatpak runtime links when not on Endless
+override_dh_link:
+
+override_dh_installdirs:
```

Comment:
I think this is redundant – AFAIK the only other "distro" that includes this package is freedesktop-sdk which doesn't use Debian packaging. Harmless, though!

Reply:
Yes, I wasn't so sure if debian or other derivative used it so added to be on the same side, can remove if you prefer.

Reply:
Harmless, let's leave it